pom.xml

```diff
@@ -609,80 +569,125 @@
                 </plugins>
             </reporting>
         </profile>
+
         <profile>
-            <id>java9</id>
```

Comment:
The `java9` and `java12` profiles disappeared entirely.  Maybe this is the only way, but I wonder whether it is possible to make a change with a smaller diff from upstream.

Reply:
was able to work around this with <?ignore ..... ?>